autoload/kite/status.vim

```diff
@@ -33,6 +33,12 @@ function! kite#status#status(...)
     return
   endif
 
+  " If previously Kite wasn't logged in / installed / running, and now it is,
+  " trigger BufEnter to activate completion etc.
+  if index(['Kite: not logged in', 'Kite: not installed', 'Kite: not running'], getbufvar(buf, 'kite_status')) != -1
```

Comment:
nit: `Kite: not logged in` is no longer a vaild status

Reply:
Do we no longer even check whether the user is logged in?

Reply:
as discussed: Remove the login check for the status line

Reply:
Done in 9a35159.